src/components/about-dialog/about-dialog.scss

```diff
@@ -0,0 +1,5 @@
+.about-dialog {
+  &__header {
+    color: black;
```

Comment:
should it be black? please look at styles of other dialogs and make it consistent -- mb we do not need this style

Reply:
It is black in other dialogs. An toolbar rewrites this color to white